test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/ResponseBodyTests.cs

```diff
@@ -515,15 +515,15 @@ public class ResponseBodyTests
                 await Assert.ThrowsAsync<TaskCanceledException>(() => responseTask);
                 Assert.True(context.DisconnectToken.WaitHandle.WaitOne(TimeSpan.FromSeconds(5)));
                 // It can take several tries before Write notices the disconnect.
-                for (int i = 0; i < 100; i++)
+                for (int i = 0; i < 1000; i++)
```

Comment:
1000 seems like a lot, why did we select it over 10, 100, or 5? Do we know specifically why it takes time (is it time or iterations?) to "notice" the disconnect, or is there maybe a more accurate thing we could wait on? I figure actually running the tests is the first step to answering these questions, but I'll bring them up now anyway.

Reply:
Looks like they were picked arbitrarily, so I'm just going to do the "safest" approach and take the maximum.

Reply:
It was based on what it took to pass reliably.